lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -13,7 +13,7 @@ class Base
           ::RuntimeError,
         ].freeze
 
-        BASE_POINTS = 1_500_000
+        BASE_POINTS = 300_000 # 30 minutes (5min = 50K)
```

Comment:
@pbrisbin I also added `HOUR_POINTS` because I think hours are easier to think about for the size of remediation here - esp when we add the severity threshold of two hours. wdyt?